server/src/main/java/org/elasticsearch/indices/analysis/AnalysisModule.java

```diff
@@ -116,7 +114,6 @@ public AnalysisRegistry getAnalysisRegistry() {
         hunspellService) {
         NamedRegistry<AnalysisProvider<TokenFilterFactory>> tokenFilters = new NamedRegistry<>("token_filter");
         tokenFilters.register("stop", StopTokenFilterFactory::new);
-        tokenFilters.register("standard", StandardTokenFilterFactory::new);
```

Comment:
this is breaking, we need to take care of backward compat

Reply:
I pushed https://github.com/elastic/elasticsearch/pull/33310/commits/67efaae12d76f658530555006a4388df863002a4